channels.yaml

```
@@ -122,3 +122,21 @@ releases:
     maxChannelServerVersion: v2.6.99
     serverArgs: *serverArgs-v1
     agentArgs: *agentArgs-v1
+  - version: v1.20.13-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v1
+    agentArgs: *agentArgs-v1
+  - version: v1.21.7-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v2
+      <<: *serverArgs-v1
+      etcd-arg:
+        type: array
+    agentArgs: *agentArgs-v1
+  - version: v1.22.4-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
```

I think this should be:
```suggestion
    minChannelServerVersion: v2.6.3-alpha1
```
cc @kinarashah @deniseschannon

---

To my understanding, setting the `minChannelServerVersion` to that will block its use in `master-head` until an actual `2.6.3*` is actually cut. This is not desirable. 

This PR is for visualization purposes anyway, I personally did not have intentions on trying to merge RC versions of K3s into KDM.

---

Right but this will probably get modified in-place for the final release and I'd like some clarity on how we restrict this to the correct 2.6 releases, assuming that's the desired behavior